src/clusterfuzz/_internal/cron/grouper.py

```diff
@@ -97,19 +97,26 @@ def _emit_grouping_event(moved_testcase: int,
           group_merge_reason=group_merge_reason))
 
 
+def _get_testcase_log_info(testcase: TestcaseAttributes):
+  """Return testcase data as string for logging."""
+  log_str = (f'group={testcase.group_id}, '
+             f'crash_type={testcase.crash_type}, '
+             f'crash_state={testcase.crash_state}, '
+             f'security_flag={testcase.security_flag}, '
+             f'one_time_crasher={testcase.one_time_crasher_flag}, '
+             f'fuzzer={testcase.fuzzer_name}, '
+             f'job={testcase.job_type}')
```

Comment:
Maybe it will be easier to read in the logs explorer with \n?

If that's the case, maybe use `textwrap.dedent` then.